bin/node/runtime/pangolin/src/pallets/bridge_grandpa.rs

```diff
@@ -0,0 +1,27 @@
+pub use pallet_bridge_grandpa::Instance1 as MillauGrandpa;
```

Comment:
```suggestion
pub use pallet_bridge_grandpa::Instance1 as WithMillauGrandpa;
```

same with bridge_dispatch (WithMillauDispatch) and bridge_messages (WithMillauGrandpa) or remove `With` in bridge_dispatch and bridge_messages